RecoLocalCalo/HcalRecAlgos/test/HcalSevLvlAnalyzer.cc

```diff
@@ -62,8 +65,11 @@ class HcalSevLvlAnalyzer : public edm::EDAnalyzer {
 // constructors and destructor
 //
 HcalSevLvlAnalyzer::HcalSevLvlAnalyzer(const edm::ParameterSet& iConfig)
-
-{
+    :
+#ifdef THIS_IS_AN_EVENTSETUP_EXAMPLE
+      totkSet_(esConsumes()),
+#endif
```

Comment:
Now that the event setup happens with the `getData` method, I believe these lines are not needed anymore

Reply:
removed